control-plane/firestore.rules

```diff
@@ -7,7 +7,7 @@ service cloud.firestore {
   match /dabases/{database}/documents {
     match /{document=**} {
       allow read, write: if
-          request.time < timestamp.date(2022, 7, 21);
+          request.time < timestamp.date(2022, 7, 30);
```

Comment:
We should remove this and have it open for all to write for now since this fixed date potentially breaks the demo for anyone checking it out after 7/30/22 and generating data after that date.